source/common/router/config_impl.h

```diff
@@ -142,8 +142,9 @@ class VirtualHostImpl : public VirtualHost {
   std::vector<VirtualClusterEntry> virtual_clusters_;
   SslRequirements ssl_requirements_;
   const RateLimitPolicyImpl rate_limit_policy_;
-  const ConfigImpl& global_route_config_;
   CorsPolicySharedPtr cors_policy_;
+  const ConfigImpl& global_route_config_; // See note in RouteEntryImplBase::clusterEntry() on why
+                                          // raw ref to the top level config is currently safe.
```

Comment:
nit :mild preference to bump this to the line above the member variable rather than having a multi-line comment here and below

Reply:
I agree! This comment came in when I merged master.

Reply:
Oh weird, I must have read the diffs wrong.  Please disregard!